app/uploaders/achievement_badge_uploader.rb

```diff
@@ -0,0 +1,13 @@
+# Image uploader for Achievement Badge
+# This class contains achievement badge specific settings for the image uploader.
+class AchievementBadgeUploader < ImageUploader
```

Comment:
The context was I tried extending the `ImageUploader` class, but it ended up implementing the `default_url` for `User#icon`. The options were to either handle this default using AchievementHelper, or do it in the Uploader.